experimental/dds/tree2/src/test/shared-tree/fuzz/anchorStability.fuzz.spec.ts

```diff
@@ -82,4 +72,44 @@ describe("Fuzz - anchor stability", () => {
 			emitter,
 		});
 	});
+	describe("Anchors are stable", () => {
+		// TODO: Add deletes once anchors are stable across removal
+		// TODO: Add moves once we have a generator for them
+		const editGeneratorOpWeights: Partial<EditGeneratorOpWeights> = {
+			insert: 2,
+			undo: 1,
+			redo: 1,
+			synchronizeTrees: 1,
+		};
+		const generatorFactory = () =>
+			takeAsync(opsPerRun, makeOpGenerator(editGeneratorOpWeights));
+		const generator = generatorFactory() as AsyncGenerator<Operation, AbortFuzzTestState>;
+		const model: DDSFuzzModel<
+			SharedTreeTestFactory,
+			Operation,
+			DDSFuzzTestState<SharedTreeTestFactory>
+		> = {
+			workloadName: "SharedTree",
+			factory: new SharedTreeTestFactory(onCreate),
+			generatorFactory: () => generator,
+			reducer: fuzzReducer,
+			validateConsistency: () => {},
+		};
+
+		let anchors: Map<Anchor, [UpPath, Value]>;
+		const emitter = new TypedEventEmitter<DDSFuzzHarnessEvents>();
+		emitter.on("testStart", (initialState: AbortFuzzTestState) => {
+			anchors = createAnchors(initialState.clients[0].channel);
+		});
+
+		emitter.on("testEnd", (finalState: AbortFuzzTestState) => {
+			validateAnchors(finalState.clients[0].channel, anchors, false);
+		});
+
+		createDDSFuzzSuite(model, {
+			defaultTestCount: runsPerBatch,
+			numberOfClients: 2,
+			emitter,
+		});
```

Comment:
It looks like we are only validating the anchors for the one tree, but there are 2 clients. Should we validate the anchors for this other tree as well?